lib/chat_api/twilio/notification.ex

```diff
@@ -10,23 +10,26 @@ defmodule ChatApi.Twilio.Notification do
     Conversations.Conversation,
     Customers.Customer,
     Twilio,
+    Twilio.TwilioAuthorization,
     Messages.Message
   }
 
   @spec notify_sms(ChatApi.Messages.Message.t()) :: {:error, any} | {:ok, Tesla.Env.t()}
   def notify_sms(%Message{
         conversation_id: conversation_id,
         body: body,
-        account_id: account_id,
-        customer: %Customer{phone: customer_phone}
```

Comment:
just curious - looks like most of the refactor here was in order to move this into the `with`. Is that becaue we can't be sure that the `customer` is preloaded? @reichert621

Reply:
in this case, the message is coming from the agent/user, not the customer -- so `customer` here will be `nil`... we need to get the `customer` from the message's `conversation` instead :)